src/dbc_content.rs

```diff
@@ -67,6 +68,14 @@ impl AmountSecrets {
         }
     }
 
+    pub fn to_pedersen_commitment(&self) -> RistrettoPoint {
+        PedersenGens::default().commit(Scalar::from(self.amount), self.blinding_factor)
+    }
+
+    pub fn to_pedersen_commitment_compressed(&self) -> CompressedRistretto {
+        self.to_pedersen_commitment().compress()
+    }
+
```

Comment:
I think just `to_pedersen_commitment()` is enough, if someone wants the Compressed point, they can compress it themselves

Reply:
done.